src/lib/booleans.js

```diff
@@ -0,0 +1,121 @@
+function negate(a) {
+  if (a === true) {
+    return false;
+  }
+  return true;
+}
+
+function both(a, b) {
+  if (a && b === true) {
+    return true;
+  }
+  return false;
+}
+
+function either(a, b) {
+  if (a || b === true) {
+    return true;
+  }
+  return false;
+}
+
+function none(a, b) {
+  if (!a && !b) {
+    return true;
+  }
+  return false;
+}
+
+function one(a, b) {
+  if ((a && !b) || (!a && b)) {
+    return true;
+  }
+  return false;
+}
+
+function truthiness(a) {
+  return Boolean(a);
+}
+
+function isEqual(a, b) {
+  if (a === b) {
+    return true;
+  }
+  return false;
+}
+
+function isGreaterThan(a, b) {
+  if (a > b) {
+    return true;
+  }
+  return false;
+}
+
+function isLessThanOrEqualTo(a, b) {
+  if (a <= b) {
+    return true;
+  }
+  return false;
+}
+
+function isOdd(a) {
+  if (a % 2) {
+    return true;
+  }
+  return false;
+}
+
+function isEven(a) {
+  if (a % 2 === 0) {
+    return true;
+  }
+  return false;
+}
```

Comment:
All of these could be simplified by returning just the result of the expression. Rather than adding a solution for all of these as I think youve already had these functions reviewed in the original js basics repo (which ideally would have been updated before stating this one) Ill defer to the comments on that one and just focus on express related issues as theres not much point in me repeating myself

If this isnt the case and you never got these then do please let me know if you have any questions about how to solve these things as its important you understand the basics